corehq/apps/reports/templates/reports/filters/form_app_module_drilldown.html

```diff
@@ -89,4 +89,11 @@
             {% include 'reports/filters/partials/fuzzy_checkbox.html' %}
         </div>
     {% endif %}
+
+    {% if all_form_retrieval_failed %}
+        <div class="alert alert-warning" data-bind="visible: show" style="margin-top: 1em;">
+          {% trans "Filtering by unknown forms is currently unavailable for this project." %}
+          {% trans "If the problem persists please report an issue." %}
```

Comment:
Looks good, but maybe the error message should indicate that this is expected behavior so it doesn't get reported a ton?

Reply:
This is only expected for ICDS and those will just be internal users and are unfortunately used to scale problems in reports.

This would be a real issue for anyone else and they should definitely report an issue.

Reply:
cool, makes sense